tls/s2n_security_policies.c

```diff
@@ -1367,6 +1343,13 @@ struct s2n_security_policy_selection security_policy_selection[] = {
     { .version = NULL, .security_policy = NULL, .ecc_extension_required = 0, .pq_kem_extension_required = 0 }
 };
 
+const char *deprecated_security_policies[] = {
+    "PQ-SIKE-TEST-TLS-1-0-2019-11",
+    "PQ-SIKE-TEST-TLS-1-0-2020-02",
+};
+
+const size_t deprecrated_security_policies_len = s2n_array_len(deprecated_security_policies);
```

Comment:
Nit: These are closely related enough that I'd probably keep them closer together. But it doesn't really matter much.
```suggestion
};
const size_t deprecrated_security_policies_len = s2n_array_len(deprecated_security_policies);
```

Reply:
Fixed